app/views/admin/subscriptions/index.html.haml

```diff
@@ -6,7 +6,7 @@
 
 - content_for :page_actions do
   %li{ "data-controller": "help-modal-link", "data-action": "click->help-modal-link#open", "data-help-modal-link-target-value": "new_subscription_help_modal" }
-    %a.button.icon-plus#new-subscription
+    %a.button.icon-plus#new-subscription{ href: "javascript:void(0)" }
```

Comment:
Not sure to understand this change?

Reply:
The capybara spec was unable to target the link.

Reply:
We shouldn't change the code for no other benefit than making a spec work. I could imagine that Capybara expects an `href` to use `click_link`. But could you use `find(...).click`?